src/main/java/com/laytonsmith/core/functions/Trades.java

```diff
@@ -198,7 +198,9 @@ public String getName() {
 		@Override
 		public String docs() {
 			return "array {} Returns an array where the keys are currently registered merchant IDs and the values are"
-					+ " the corresponding window titles of those merchants.";
+					+ " the corresponding window titles of those merchants. For entity merchants,"
+					+ " this will be the custom name if it has one, falling back to Profession for Villagers,"
+					+ " and entity type for any other trading entity.";
 		}
```

Comment:
I'm confused about this. As far as I can tell, virtual merchants are never entity merchants. So what is this new documentation describing?